packages/xdl/src/Binaries.ts

```diff
@@ -54,15 +50,15 @@ export async function addToPathAsync(name: string) {
   _prependToPath(binariesPath);
 }
 
-function _expoRCFileExists() {
+function _expoRCFileExists(): boolean {
   try {
     return fs.statSync(path.join(UserSettings.dotExpoHomeDirectory(), 'bashrc')).isFile();
   } catch (e) {
     return false;
   }
 }
 
-function _prependToPath(newPath) {
+function _prependToPath(newPath: string): void {
```

Comment:
`: void` is redundant. In my opinion, there isn't any reason we should declare return types explicitly if TypeScript can do it for us. I think we already discussed that but previously you were pointing out that it may be helpful for functions we export from modules but it's not the case here.
The same applies to the other return types of the functions in this file.

Reply:
On one hand, it's true that these types can usually inferred by the compiler and when not, we get an error (because we have `noImplicitAny` set) and can then add the return type in that case. The inferred types are also visible in any editor that has a TypeScript support installed.

On the other hand, those inferred types are not visible in every context where the code is read, for example when reviewing/browsing it on GitHub.

I don't personally have a strong opinion about this, one way or the other, but I'd like to hear others' opinions (particularly @ide, since he was the one suggesting that I add the return types) and then come up with a rule of thumb when to define return types explicitly and when not.